COMETwebapp/Components/SubscriptionDashboard/SubscribedTable.razor

```diff
@@ -78,6 +78,12 @@ Copyright (c) 2022 RHEA System S.A.
                                 var lastParameter = (ParameterValueSet)associatedParameterValueSet.Revisions.Last().Value;
                                 subscribedLastActualValue = lastParameter.ActualValue;
                             }
+
+                            var evolution = (double)0;
+                            if(actualValue?.Count == 1 && lastActualValue?.Count == 1 && !actualValue.First().Equals("-") && !lastActualValue.First().Equals("-") && !lastActualValue.First().Equals("0"))
```

Comment:
For readability, you could add line breaks when the line is very long.
if(actualValue?.Count == 1 
    && lastActualValue?.Count == 1 
    && !actualValue.First().Equals("-")
    && !lastActualValue.First().Equals("-") 
    && !lastActualValue.First().Equals("0"))